proposed/config-meta.md

```diff
@@ -0,0 +1,134 @@
+# Config Meta Document
+
+## 1. Introduction
+This document describes the process and discussions that led to the Config PSR. It's goal is to explain the reasons 
+behind each decision.
+
+The word `Container` in this document is to be interpreted as the `ContainerInterface` of the PSR Container proposal.
+
+## 2. Why bother?
+The configuration of instances via factories is not uniform, uses different ways to retrieve and check the needed 
+options and it's not clear which configuration options needs the factory to create the instance.
+
+## 3. Scope
+
+### 3.1 Goals
+The goal set by the Config PSR is to standardize how factories uses a configuration to create instances, support for 
+auto discovery of needed configuration, to reduce boilerplate code and to make it more readable and easier to understand. 
+It can also be used to build the name of the *Container* entry.
+
+### 3.2 Non-goals
+* It's not a goal to define a whole project/library configuration structure.
+* It's not a goal to validate the configuration option values.
```

Comment:
Yes, it validates the configuration structure depending on the implemented interfaces.